src/test/com/yetanalytics/datasim/xapi/profile/template/rule_test.clj

```diff
@@ -1,113 +1,96 @@
 (ns com.yetanalytics.datasim.xapi.profile.template.rule-test
-  (:require
-   [clojure.test :refer :all]
-   [com.yetanalytics.datasim.xapi.profile.template.rule
-    :refer [parse-rule apply-rules-gen follows-rule?] :as r]
-   [com.yetanalytics.datasim.input :as input]
-   [clojure.java.io :as io]
-   [cheshire.core :as json]
-   [clojure.spec.alpha :as s]
-   [xapi-schema.spec :as xs]))
+  (:require [clojure.test :refer [deftest testing is]]
+            [clojure.java.io :as io]
+            [cheshire.core :as json]
+            [clojure.spec.alpha :as s]
+            [xapi-schema.spec :as xs]
+            [com.yetanalytics.datasim.xapi.profile.template.rule :as r]
+            [com.yetanalytics.datasim.test-fixtures :as fix]))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Fixtures
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(def gen-seed 42)
+
+(def example-statement
+  {"id"        "59de1b06-bb6c-4708-a51a-b3d403c491db",
+   "actor"     {"name" "Alice Faux"
+                "mbox" "mailto:alice@example.org"},
+   "verb"      {"id" "https://adlnet.gov/expapi/verbs/launched"},
+   "object"    {"id"         "https://example.org/career/1054719918",
+                "definition" {"type" "https://w3id.org/xapi/tla/activity-types/career"},
+                "objectType" "Activity"},
+   "context"   {"registration" "d7acfddb-f4c2-49f4-a081-ad1fb8490448"},
+   "timestamp" "2021-03-18T17:36:22.131Z"})
+
+(def example-rule
+  {:presence "excluded",
+   :location [#{"context"} #{"contextActivities"}]})
+
+(def group-as-actor-rules
+  [;; TODO: Right now this will only apply the second rule if the first is present
+   ;; the path->spec function can't resolve ["actor" "member"]
+   {:location "$.actor.objectType"
+    :presence "included"
+    :all      ["Group"]}
+   {:location "$.actor.member[0]"
+    :presence "included"
+    :all      [{"mbox"       "mailto:milt@yetanalytics.com"
+                "name"       "milt"
+                "objectType" "Agent"}]}])
 
 ;; we can pull some actual rules from cmi5
-(def cmi5-templates
-  (:templates
-   (input/from-location :profile :json "dev-resources/profiles/cmi5/fixed.json")))
+(def cmi5-templates (:templates fix/cmi5-profile))
 
-(def all-rules
-  (mapcat :rules cmi5-templates))
+(def all-rules (mapcat :rules cmi5-templates))
```

Comment:
Do these rules cover all the cases we looked at in the huddle the other day?

Reply:
Nope I want to do that in the _next_ branch.

Reply:
I had a feeling, so this one is just organizational.

Reply:
Yeah pretty much.